app/uk/gov/hmrc/controllers/DocumentationController.scala

```diff
@@ -17,12 +17,13 @@
 package uk.gov.hmrc.controllers
 
 import controllers.Assets
+
 import javax.inject.{Inject, Singleton}
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
-import uk.gov.hmrc.play.bootstrap.controller.FrontendBaseController
+import uk.gov.hmrc.play.bootstrap.controller.BackendController
 
 @Singleton
-class Documentation @Inject()(assets: Assets, cc: MessagesControllerComponents) extends FrontendBaseController {
+class Documentation @Inject()(assets: Assets, cc: MessagesControllerComponents) extends BackendController(cc) {
```

Comment:
Siva, why replacing FrontendBaseController with FrontendBaseController?

Reply:
oh you mean, BackendController ? bootstrap play 2.8 libraries were split into frontend and backend specific now and only one lib is allowed to use. This service is a backend one and hence used BackendController to replace the Frontend one.

Reply:
oh sorry, yah i mean BackendController.. ok then fine. thanks